core/src/components/select/test/single-value/index.html

```diff
@@ -133,6 +133,16 @@
         </ion-select>
       </ion-item>
 
+      <ion-item>
```

Comment:
The `Statuses` alert select looks like it already has a disabled button in its test, so not sure if this test gives us any additional coverage.